### ISSUE TITLE
The zotero reader breaks while loading a corpus.

### ISSUE THREAD

**Issue by nischalsamji** [diging/tethne#80]
When trying to import a corpus generated from a corpus getting the following error. The folder contains a rdf exported from zotero.

``` python
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/nischal/diging/tethne/tethne/readers/zotero.py", line 177, in read
    papers = ZoteroParser(path).parse()
  File "/home/nischal/diging/tethne/tethne/readers/base.py", line 278, in parse
    self.handle(tag, o)    
  File "/home/nischal/diging/tethne/tethne/readers/base.py", line 287, in handle
    data = handler(data)
  File "/home/nischal/diging/tethne/tethne/readers/zotero.py", line 86, in handle_date
    return datetime.strptime(str(value), "%m/%d/%Y").date().year
  File "/usr/lib/python2.7/_strptime.py", line 325, in _strptime
    (data_string, format))
ValueError: time data 'December 23, 2014' does not match format '%m/%d/%Y'
```

**Comment by nischalsamji** [comment]
Updated the branch bugfix- #80 with a fix. Testing in progress.

**Comment by nischalsamji** [comment]
Fixed the Issue bugfix-#80 merged with develop
